tekton/runs/nb-test-run.yaml

```diff
@@ -15,34 +15,44 @@ spec:
   - name: test-target-name
     value: manual-testinig
   - name: artifacts-gcs
-    value: gs://kubeflow-ci-deployment/gabrielwen-testing-2
+    value: gs://kubeflow-ci_temp/jlewi_mnist_testing/2020-0619
   - name: junit-path
     value: artifacts/junit_manual-testing
   - name: testing-cluster-pattern
     value: 'kf-vbp-.*'
   - name: testing-cluster-location
     value: 'us-central1'
-  - name: output-workspace
-    value: /workspace/outputs
+  - name: notebook-output
+    value: gs://kubeflow-ci-deployment_ci-temp/mnist_test
+  - name: notebook-path
+    value: mnist/mnist_gcp.ipynb
   resources:
   # This pull number has changes blocked on:
   # https://github.com/kubeflow/testing/pull/622
-  - name: examples-repo
+  - name: notebook-repo
     resourceSpec:
       type: git
       params:
       - name: url
         value: https://github.com/kubeflow/examples.git
       - name: revision
-        value: refs/pull/803/head
+        value: master
   - name: testing-repo
     resourceSpec:
       type: git
       params:
       - name: url
-        value: https://github.com/kubeflow/testing.git
+        value: https://github.com/jlewi/testing.git
```

Comment:
Should this point to Kubeflow?

Reply:
testing-repo shouldn't be a resource anymore.